.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Unfortunately you can't use comments in JSON files (yes, I know).

Reply:
Luckily VSCode [doesn't stick to the JSON spec](https://code.visualstudio.com/Docs/languages/json#_json-with-comments) here :)